db/re/item_db_usable.yml

```diff
@@ -11011,7 +10944,7 @@ Body:
       NoAuction: true
     Script: |
       specialeffect2 EF_HEAL3;
-      bonus_script "{ bonus2 bHPRegenRate,(MaxHp/100*6),3000; }",600;
+      sc_start4 SC_G_LIFEPOTION,600000,-6,3,0,0;
```

Comment:
Do you know if the effect stacks with `SC_L_LIFEPOTION` `SC_S_LIFEPOTION` `SC_M_LIFEPOTION` on KRO?